yang/vyatta-protocols-frr-isis-v1.yang

```diff
@@ -60,7 +63,7 @@ module vyatta-protocols-frr-isis-v1 {
     The YANG module package for vyatta-protocols-frr-isis-v1";
 
   revision 2020-11-17 {
-    description "Add IPv6 ISIS interface config option";
```

Comment:
Please keep this line and add your entry on a new line. For example, something like:
```
 description "Add IPv6 ISIS interface config option.
    Added ISIS topology and mpls-te configuration.";
```